libvcx/src/api/utils.rs

```diff
@@ -267,59 +141,6 @@ pub extern fn vcx_set_next_agency_response(message_index: u32) {
     AgencyMock::set_next_response(message);
 }
 
-/// Retrieve messages from the Cloud Agent
```

Comment:
Shouldn't `vcx_messages_download` be deprecated also?

Reply:
we need to migrate mobile first